### ISSUE TITLE
I'm interested in maintaining this project

### ISSUE THREAD

**Issue by cforain** [yardteam/rrr3d#9]
First of all, thank you very much for not only creating such and amazing game, but to go to the extra mile of sharing it with the world. Really, I can't thank you enough.

Although, if it is not much to ask, is there any kind of documentation. Simple stuff such as how to build, list of dependencies and such. I know I may be asking too much, but it is worth a try.  Thank you once again.

**Comment by lexert** [comment]
Hello. Thanks for appreciation of our work. Unfortunately there is no such stuff as documentation. But i am still thinking that it is not so difficult to learn it by himself. Off course it can take some effort, but i am sure experienced developer can managed it. And if there will be some questions i can try to answer them.

Consider for example a building process. A only external dependency that is not present in the game is the boost library (because of its size). It would be enough to put the whole library into the following directory
https://github.com/yardteam/rrr3d/tree/master/prog/Third%20Party/include/boost
And that is it. The next step is to open a solution and start compiling. After compilator finishes its work, all produced dll and exe files can be copied into the folder with the game release (downloaded from steam for example)

May be there will be some issues with using exporter for the game models. But i believe i will be able to help with it too. Also you may ask questions about source code, i will try to answer them.

**Comment by cforain** [comment]
Thanks. I'm kinda lost. Although I am an experienced developer, I have very little experience in game development. Is this a visual Studio project? Thanks.

**Comment by lexert** [comment]
Yes, natively VS 2008 was used. But VS 2012 was checked as well. RRR3d.sln can be found in the root directory.

**Comment by cforain** [comment]
I'm trying to open it on Linux with visual-studio-code. I have some more questions about the physics and IA, when I get to that point. Is there a easier way of reaching you without polluting the github (like an e-mail or something?) mine is claudioforain at gmail dot com . If not, I totally understand. Thanks.

**Comment by cforain** [comment]
Sorry to bother again, but I was able to open the project, mas I'm still kinda lost. Is there any kind of lib/engine you used, or did you write everything from scratch?

Where does the code actually "starts" (like the main funcion on a C program) ? Thanks in advance once again.
